### ISSUE TITLE
Is there a chance this could be made for vanilla JS instead of react?

### ISSUE THREAD

**Issue by hscanlan** [NoahZinsmeister/web3-react#373]
Just curious - it would be great as our project doesn't use React :(

**Comment by NoahZinsmeister** [comment]
yes! web3-react@store specifically uses vanilla zustand stores to fully support non-React usage! first, i recommend checking out https://github.com/pmndrs/zustand#using-zustand-without-react. then, you might want to look at web3-react@core a bit to see how the vanilla store is wrapped for use in React, to get a sense of what you might want to do in the context of another framework.

going to have some docs on this eventually, but for now going to convert this to a discussion, feel free to ask more questions
